bindings/dotnetcore/inc/dotnetcore_common.h

```diff
@@ -7,7 +7,15 @@
 #include "broker.h"
 #include "module.h"
 
-typedef int(*coreclr_initialize_ptr)(const char* exePath,
+#if WIN64
```

Comment:
Good Catch! Thanks! Fixed.

---
In reply to: [107037679](https://github.com/Azure/azure-iot-gateway-sdk/pull/171#discussion_r107037679) [](ancestors = 107037679)